src/scopeHandler.js

```diff
@@ -79,7 +81,6 @@ export function createScopeHandler(unsafeRec, safeGlobal, sloppyGlobals) {
       return undefined;
     },
 
-    // eslint-disable-next-line class-methods-use-this
```

Comment:
Thanks for catching this. Does anyone have an automated detection of `eslint-disables-*`s that are not actually suppressing any warnings?